grammar.js

```diff
@@ -274,8 +274,7 @@ module.exports = grammar({
       seq(
         $._indent,
         optional(seq($.shebang, $._newline)),
-        repeat(choice(seq($.recipe_line, $._newline), $._newline)),
-        $._newline,
+        repeat1(seq($.recipe_line, $._newline)),
```

Comment:
This was needed so that non-newline separated recipes would have comments identified correctly:

```justfile
# foo
foo:
  echo "foo"
# bar
bar:
  echo "bar"
```

Reply:
Awesome, thank you. Would you mind splitting this change to a separate PR with a new test? Just so I can merge it right away (if not, I can split it off)

Reply:
https://github.com/IndianBoy42/tree-sitter-just/pull/119